acai-ios/Acai.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  Acai.swift
+//  acai-ios
+//
+//  Created by Artesia Ko on 3/5/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class Acai {
+    
+    static let menuItemHeaderImageIdentifier: String = "menuItemHeaderImageIdentifier"
+    static let menuItemButtonCollectionIdentifier: String = "menuItemButtonCollectionIdentifier"
+    static let menuItemCustomizationOptionsIdentifier: String = "menuItemCustomizationOptionsIdentifier"
+    static let menuItemCheckoutListIdentifier: String = "menuItemCheckoutListIdentifier"
+    
+    static let avenirNextMedium: UIFont = UIFont(name: "AvenirNextCyr-Medium", size: 12)!
+    static let avenirNextDemi: UIFont = UIFont(name: "AvenirNextCyr-Demi", size: 12)!
```

Comment:
I would make these fonts in a `UIFont+.swift` and add them as extensions of `UIFont`. similar to what @drewsdunne said about the colors